### PR TITLE
add exception message

### PR DESCRIPTION
**Pull request by qiuming520**
Signed-off-by: qiuming520 <qiuming_yewu@cmss.chinamobile.com>

### CODE REVIEW

File: pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -100,10 +100,10 @@ func (cfg *Config) Complete() CompletedConfig {
 
 func (config completedConfig) New() (*ClusterPediaServer, error) {
 	if config.ClientConfig == nil {
-		return nil, errors.New("")
+		return nil, errors.New("ClientConfig is missing")
 	}
 	if config.StorageFactory == nil {
-		return nil, errors.New("")
+		return nil, errors.New("StorageFactory config is missing")
```
Comment (Iceber) [inline pkg/apiserver/apiserver.go:106]: We could use the same error style as `genericapiserver`, after all, they are functions with the same meaning.
https://github.com/clusterpedia-io/clusterpedia/blob/01ad71bfa087b18e16eb6c981c874c023c91acff/vendor/k8s.io/apiserver/pkg/server/config.go#L592-L601

eg. `apiserver.New() called with config.ClientConfig == nil`
> `Clusterpediaapiserver.New` and  `APIServer.New` don't look good, we can just use the `apiserver.New`

**Review by Iceber — COMMENTED** [review]
Thanks for the fix, just the error message looks like it could be slightly optimized

File: pkg/apiserver/apiserver.go (hunk 1 above)
Comment (qiuming520) [reply pkg/apiserver/apiserver.go]: I have revised it as suggested .

**Comment by Iceber** [comment]
LGTM, Please rebase commits into one, because you are actually doing one thing.
> In order to try to merge in a uniform way, we don't use `squash and merge`

File: hack/install-golangci-lint.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# do proxy setting in China mainland
+if [[ -n ${CHINA_MAINLAND:-} ]]; then
+  export GOPROXY=https://proxy.golang.com.cn,direct # set domestic go proxy
+fi
```
Comment (Iceber) [inline hack/install-golangci-lint.sh:10]: In China, the developers usually configure this environment variable, and it should not be configured by the script, after all, maybe the user has their own defined `GOPROXY` that cannot override the user environment's proxy.
What do you think?

Comment (qiuming520) [reply hack/install-golangci-lint.sh:10]: yeah！ Your point is right. I have revoked this PR
Just it's easy to ignore proxy settings when installing in a new environment.

**Pull request closed without merging**